### DISCUSSION TITLE
Using QSPI Flash storage with STM32F7DISC

### DISCUSSION THREAD

**Discussion by matthgyver** [micropython/micropython#9306] [Hardware & Peripherals]
Hi,

I just tried micropython on STM32F746 Disco board and it's work great but I have not access to the integrated QSPI flash storage.
How can I run my script from this memory please ?

Thank's

**Comment by jimmo** [comment]
You will need to define the QSPI and block device config in https://github.com/micropython/micropython/tree/master/ports/stm32/boards/STM32F7DISC and compile the firmware (and maybe send a PR 
 once you're done :) ).

See for example the F769 discovery board (https://github.com/micropython/micropython/tree/master/ports/stm32/boards/STM32F769DISC) which already does this.
